.github/workflows/ci.yml

```diff
@@ -13,6 +13,12 @@ jobs:
         go-version: 1.15
       id: go
 
+    - name: Set up qemu
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up buildx
+      uses: docker/setup-buildx-action@v1
+
```

Comment:
setup buildx for multi-arch docker build

https://github.com/docker/setup-buildx-action